nomenclature/region_mapping_models.py

```diff
@@ -134,6 +137,12 @@ def all_regions(self) -> List[str]:
         cr_list = [x.name for x in self.common_regions or []]
         return nr_list + cr_list
 
+    @property
+    def renaming_dict(self) -> Dict[str, str]:
```

Comment:
I think the name `rename_mapping` more intuitive...

Reply:
Done